tracks/javascript/exercises/resistor-colors/mentoring.md

```diff
@@ -78,7 +78,10 @@ Approve the solution, but challenge them to solve it using `map`, as listed belo
 
 ### Common suggestions
 
-Here are a few common suggestions based on student's solutions. You may always suggest they re-use code from their
+Here are a few common suggestions based on student's solutions. You may always suggest they re-
+
+
+code from their
```

Comment:
This needs fixing - `re-use code from their solution`

Reply:
I'm weeping because I don't even know HOW this got like this.